### PR TITLE
Update _redirects

### PR DESCRIPTION
**Pull request by rbellary-vi**
Testing

### CODE REVIEW

**Review by rbellary-vi — COMMENTED** [review]
reverting

**Review by rbellary-vi — COMMENTED** [review]
testing

**Review by rbellary-vi — COMMENTED** [review]
testing

**Review by rbellary-vi — COMMENTED** [review]
test

**Review by rbellary-vi — COMMENTED** [review]
test

**Review by rbellary-vi — COMMENTED** [review]
update

**Review by rbellary-vi — COMMENTED** [review]
update

File: static/_redirects [hunk 1]
```diff
@@ -1,8 +1,9 @@
+https://docs.metricly.com/* https://docs.virtana.com//:splat 301!
+/integrations/agents/linux-agent/ https://docs.virtana.com/en/linux-agent.html 301!
```
Comment (TheConnMan) [inline static/_redirects:2]: This line is perfect.

File: static/_redirects [hunk 2]
```diff
@@ -1,8 +1,9 @@
+https://docs.metricly.com/* https://docs.virtana.com//:splat 301!
```
Comment (TheConnMan) [inline static/_redirects:1]: This line I think will break all the lines below once you merge it in.

**Review by rbellary-vi — COMMENTED** [review]
test

File: static/_redirects (hunk 2 above)
Comment (rbellary-vi) [reply static/_redirects]: removed the line

File: static/_redirects (hunk 1 above)
Comment (rbellary-vi) [reply static/_redirects:1]: Thank You

**Review by rbellary-vi — COMMENTED** [review]
updated New URL's

**Comment by TheConnMan** [comment]
Wonderful! If you're ready for review please add @jpadavala-vi and @germankovacevic as reviewers.

**Comment by rbellary-vi** [comment]
> Wonderful! If you're ready for review please add @jpadavala-vi and @germankovacevic as reviewers.

@TheConnMan still another 20% is pending once that also completed will add for review

**Review by rbellary-vi — COMMENTED** [review]
@jpadavala-vi  @germankovacevic  All Redirects Completed Please have a look when you have a time

**Review by jpadavala-vi — DISMISSED** [review]
looks good hope those are all the links Rajesh.

**Review by rbellary-vi — COMMENTED** [review]
added last page

File: static/_redirects [hunk 3]
```diff
@@ -1,8 +1,270 @@
...
+/api/api-datasources/ https://docs.virtana.com/en/datasources-api.html 301!
+/api/api-ingest/ https://docs.virtana.com/en/ingest-api.html 301!
+/api/api-metrics/ https://docs.virtana.com/en/metrics-api.html 301!
+/api/api-notifications/ https://docs.virtana.com/en/notifications-api.html 301!
+/api/api-packages/ https://docs.virtana.com/en/packages-api.html 301!
+/api/api-policies/ https://docs.virtana.com/en/policies-api.html 301!
+/api/api-users/ https://docs.virtana.com/en/users-api.html 301!
+/api/api-widgets/ https://docs.virtana.com/en/widgets-api.html 301!
+/api/api-elements/ https://docs.virtana.com/en/elements-api.html 301!
 /right-sizing/ec2-recommendations   /right-sizing/reports-ec2-recommendations/
```
Comment (TheConnMan) [inline static/_redirects:264]: This should redirect to the new link that you have above:   https://docs.virtana.com/en/ec2-recommendations.html

```suggestion
/right-sizing/ec2-recommendations   https://docs.virtana.com/en/ec2-recommendations.html 301!
```

File: static/_redirects [hunk 4]
```diff
@@ -1,8 +1,270 @@
...
 /right-sizing/resource-utilization   /right-sizing/reports-resource-utilization/
 /right-sizing/reports-ec2-reservations   /purchase-planning/reports-ec2-reservations/
 /right-sizing/resource-utilization   /right-sizing/reports-ec2-recommendations/
```
Comment (TheConnMan) [inline static/_redirects:267]: This should redirect to the new link that you have above:   https://docs.virtana.com/en/ec2-recommendations.html

```suggestion
/right-sizing/resource-utilization   https://docs.virtana.com/en/ec2-recommendations.html 301!
```

File: static/_redirects [hunk 5]
```diff
@@ -1,8 +1,270 @@
...
+/api/api-ingest/ https://docs.virtana.com/en/ingest-api.html 301!
+/api/api-metrics/ https://docs.virtana.com/en/metrics-api.html 301!
+/api/api-notifications/ https://docs.virtana.com/en/notifications-api.html 301!
+/api/api-packages/ https://docs.virtana.com/en/packages-api.html 301!
+/api/api-policies/ https://docs.virtana.com/en/policies-api.html 301!
+/api/api-users/ https://docs.virtana.com/en/users-api.html 301!
+/api/api-widgets/ https://docs.virtana.com/en/widgets-api.html 301!
+/api/api-elements/ https://docs.virtana.com/en/elements-api.html 301!
 /right-sizing/ec2-recommendations   /right-sizing/reports-ec2-recommendations/
 /right-sizing/resource-utilization   /right-sizing/reports-resource-utilization/
```
Comment (TheConnMan) [inline static/_redirects:265]:
```suggestion
/right-sizing/resource-utilization https://docs.virtana.com/en/resource-utilization.html 301!
```

File: static/_redirects [hunk 6]
```diff
@@ -1,8 +1,270 @@
...
 /right-sizing/reports-ec2-reservations   /purchase-planning/reports-ec2-reservations/
```
Comment (TheConnMan) [inline static/_redirects:266]:
```suggestion
/right-sizing/reports-ec2-reservations https://docs.virtana.com/en/ec2-reservation-recommendations.html 301!
```

File: static/_redirects [hunk 7]
```diff
@@ -1,8 +1,270 @@
...
 /right-sizing/resource-utilization   /right-sizing/reports-ec2-recommendations/
 /capacity-monitoring/metrics-page/   /capacity-monitoring/metrics/
```
Comment (TheConnMan) [inline static/_redirects:268]:
```suggestion
/capacity-monitoring/metrics-page/ https://docs.virtana.com/en/metrics.html 301!
```

File: static/_redirects [hunk 8]
```diff
@@ -1,8 +1,270 @@
...
-/integrations/linux-agent   /integrations/agents/linux-agent/
 /data-visualizaiton/metrics   /capacity-monitoring/metrics/
```
Comment (TheConnMan) [inline static/_redirects:269]:
```suggestion
/data-visualization/metrics https://docs.virtana.com/en/metrics.html 301!
```

File: static/_redirects [hunk 9]
```diff
@@ -1,8 +1,270 @@
...
 /alerts-notificaitons/policies   /capacity-monitoring/policies/
```
Comment (TheConnMan) [inline static/_redirects:270]:
```suggestion
/alerts-notificaitons/policies https://docs.virtana.com/en/policies.html 301!
```

File: static/_redirects [hunk 10]
```diff
@@ -1,8 +1,270 @@
...
+/api/api-policies/ https://docs.virtana.com/en/policies-api.html 301!
+/api/api-users/ https://docs.virtana.com/en/users-api.html 301!
+/api/api-widgets/ https://docs.virtana.com/en/widgets-api.html 301!
+/api/api-elements/ https://docs.virtana.com/en/elements-api.html 301!
 /right-sizing/ec2-recommendations   /right-sizing/reports-ec2-recommendations/
 /right-sizing/resource-utilization   /right-sizing/reports-resource-utilization/
 /right-sizing/reports-ec2-reservations   /purchase-planning/reports-ec2-reservations/
 /right-sizing/resource-utilization   /right-sizing/reports-ec2-recommendations/
 /capacity-monitoring/metrics-page/   /capacity-monitoring/metrics/
-/integrations/linux-agent   /integrations/agents/linux-agent/
```
Comment (TheConnMan) [inline static/_redirects:6]: I think this needs to be left in and redirected to `https://docs.virtana.com/en/linux-agent.html 301!`.

**Review by TheConnMan — CHANGES_REQUESTED** [review]
Nice work, thanks for completing this. Few final touches then I'll approve.

**Review by TheConnMan — APPROVED** [review]

**Review by TheConnMan — APPROVED** [review]
Thanks!

**Pull request merged**